src/main/java/calc/type/Operation.java

```diff
@@ -0,0 +1,30 @@
+package calc.type;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operation {
+    PLUS("+", (a, b) -> (int) a + (int) b),
+    MINUS("-", (a, b) -> (int) a - (int) b),
+    MULTIPLY("*", (a, b) -> (int) a * (int) b),
+    DIVIDE("/", (a, b) -> (int) a / (int) b);
+
+    private String operation;
+    private BiFunction<Integer, Integer, Integer> resultFunction;
+
+    Operation(String operation, BiFunction resultFunction) {
```

Comment:
```suggestion
    Operation(String operation, BiFunction<Integer, Integer, Integer> resultFunction) {
```

BiFunction에 타입을 지정해보면 어떨까요? 🤔